react/features/dynamic-branding/functions.js

```diff
@@ -7,8 +7,8 @@
  * @param {string} path - The URL path.
  * @returns {string}
  */
-export function extractFqnFromPath(path: string) {
-    const parts = path.split('/');
+export function extractFqnFromPath() {
+    const parts = window.location.pathname.split('/');
```

Comment:
shouldn't we use the locationUrl from state? I don't think that window is available in native

Reply:
Currently dynamic-branding is used for web only. If we want to use it for mobile too we would have to make some more changes because currently the `locationUrl` is set only after we join the conference.